Next-Gen apps docs & release notes (GA: week ending 6/23) (#2587)
SumoLogic/sumologic-documentation: blog-service/2023-06-15-next-gen-apps.md

---
title: New Next-Gen Applications (Apps)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - apps
  - app catalog
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

In response to many requests for better App management, we are happy to announce the release of our new Next-Gen Apps located within a new section of the Applications Catalog. Sumo Logic's next generation of Apps introduce features not previously available within Classic Apps and simplifies the management and maintenance of Apps. 

**Key Features**

* In order to know which Apps have been installed within your account these new apps will provide a badge noting their installed status. 

> * When an update to an installed App becomes available these new apps will provide a badge noting their is an update available. An upgrade can then be applied with a simple click within the App Catalog.  

Reviewer: ```suggestion
* When an update to an installed App becomes available, these new apps will provide a badge noting there is an update available. An upgrade can then be applied with a simple click within the App Catalog.  
```

* Installed Apps can be uninstalled and removed through the App Catalog, making it easy to clean up Apps you no longer use.  

> * Additional filter options making it easy to find your installed or upgradeable Apps. 

Reviewer: ```suggestion
* Additional filter options make it easy to find your installed or upgradeable Apps. 
```

* To support the ability to upgrade and to make sure upgrades do not overwrite any customizations you may apply, the content installed by an app is immutable. If you wish to customize the content you will need to first make a custom copy of the content. 
* Role-based access controls, which ensure that only authorized individuals within your organization can install, uninstall, or update apps. To perform these actions you must be a member of the default Administrator role. Sumo Logic will be providing a new “Manage Apps” role capability soon, which can be used to grant these permissions to additonal roles.

While the initial set of new apps is designed to help you get started using the Sumo Logic distribution of the Open Telemetry agent, we haven't forgotten about those who may continue to send data through the Sumo Logic Installed Collector and Hosted Sources. We will be converting all the existing Classic Apps to the next gen framework in the coming months and will continue to provide support for these collection methods. In the meantime, our Classic Apps will remain available to you within the Application Catalog.

> Over the coming year Sumo Logic will also be making additional improvements within the Applications Catalog to assist with better search, filtering, upgrade notifications, and management. Please lookout for these future updates. 

Reviewer 2: ```suggestion
Over the coming year, Sumo Logic will be making additional improvements within the Apps Catalog to assist with better search, filtering, upgrade notifications, and management. Stay tuned for these future updates. 
```
